File: shoc-containerize/src/main/java/io/imast/shoc/containerize/EngineInstanceInfo.java
package io.imast.shoc.containerize;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * The engine instance
 * 
 * @author davitp
 */
@Data
@Builder(toBuilder = true)
@NoArgsConstructor
@AllArgsConstructor
public class EngineInstanceInfo {
 
    /**
     * The engine id
     */
    private String id;
    
    /**
     * The engine name
     */
    private String name;
    
    /**
     * The engine driver
     */
    private String driver;
    
    /**
     * Indicates if engine is in running state
     */
    private boolean running;
    
    /**
     * The number of images in engine
     */
    private Integer images;
    
    /**
     * The number of containers in engine
     */
    private Integer containers;
    
}

File: Shoc.Engine.Model/EngineInstanceInfo.cs
namespace Shoc.Engine.Model
{
    /// <summary>
    /// The engine instance info
    /// </summary>
    public class EngineInstanceInfo
    {
        /// <summary>
        /// The identifier of engine instance
        /// </summary>
        public string Id { get; set; }

        /// <summary>
        /// The name of engine instance
        /// </summary>
        public string Name { get; set; }

        /// <summary>
        /// The driver of engine instance
        /// </summary>
        public string Driver { get; set; }

        /// <summary>
        /// The flag to indicate if engine is in running state
        /// </summary>
        public bool Running { get; set; }

        /// <summary>
        /// The number of images in engine instance
        /// </summary>
        public long? Images { get; set; }

        /// <summary>
        /// The number of containers running in engine instance
        /// </summary>
        public long? Containers { get; set; }
    }
}